packages/jellyfish-api-core/__tests__/category/account/sendTokensToAddress.test.ts

```diff
@@ -0,0 +1,136 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { SelectionModeType, SendTokensOptions } from '../../../src/category/account'
+
+describe('SendTokenToAddress', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await setup()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  let addressA: string
+  let addressB: string
+  let addressC: string
+  let tokenAddress: string
+
+  async function setup (): Promise<void> {
+    tokenAddress = await container.call('getnewaddress')
+    addressA = await container.call('getnewaddress')
+    addressB = await container.call('getnewaddress')
+    addressC = await container.call('getnewaddress')
+
+    await createToken(tokenAddress, 'CAT', 100)
+    await createToken(tokenAddress, 'ETH', 100)
+    await createToken(tokenAddress, 'DETH', 100)
+
+    await accountToAccount('CAT')
+    await accountToAccount('ETH')
+    await accountToAccount('DETH')
+  }
+
+  async function accountToAccount (symbol: string): Promise<void> {
+    await container.call('accounttoaccount', [tokenAddress, { [addressA]: `${20}@${symbol}` }])
+    await container.call('accounttoaccount', [tokenAddress, { [addressB]: `${30}@${symbol}` }])
+    await container.call('accounttoaccount', [tokenAddress, { [addressC]: `${50}@${symbol}` }])
+    await container.generate(1)
+  }
```

Comment:
this code messed up the test context, as mentioned, you only need 1 coin to test the selectionMode
don't add another unrelated fixtures
```
await accountToAccount('CAT')
await accountToAccount('ETH')
await accountToAccount('DETH')
```

Reply:
One coin will have side effects. I tried doing one coin, but having to run multiple test cases will make it hard to assert the exact balances before and after tokens are sent. @jingyi2811 told me make sure tests are passing independently and also the whole block. Doing one coin will make the tests fail if run independently.

Reply:
For example, it i run a test to check for PIE and deduct 10 coin and then test to check crumbs , obviously the balances will be different based on whether the test is run in block (the whole suite) or independently (just the it)

Reply:
try this
```
setup() {
    addrA = await container.call('getnewaddress')
    addrB = await container.call('getnewaddress')
    addrC1 = await container.call('getnewaddress')
    addrC2 = await container.call('getnewaddress')
    addrC3 = await container.call('getnewaddress')
    
    const tokenAddrA = await container.call('getnewaddress')
    const tokenAddrB = await container.call('getnewaddress')
    const tokenAddrC = await container.call('getnewaddress')
    await createToken(tokenAddrA, 'ANT', 200)
    await createToken(tokenAddrB, 'BAT', 200)
    await createToken(tokenAddrC, 'CAT', 200)
    
    await container.call('acc2acc', [tokenAddrA, {[addrA]:`${3}@ANT`}])
    await container.call('acc2acc', [tokenAddrB, {[addrB]:`${68}@BAT`}])
    await container.call('acc2acc', [tokenAddrC, {[addrC1]:`${89}@CAT`}])
    await container.call('acc2acc', [tokenAddrC, {[addrC2]:`${15}@CAT`}])
    await container.call('acc2acc', [tokenAddrC, {[addrC3]:`${54}@CAT`}])
    await container.generate(1)
}

Reply:
Thanks for the setup. It may work if we wont be testing PIE and CRUMBS. We can use CAT to test both selection modes but due to side effects it will won't work. The problem is that we can not use the same coin to test and assert balances which in what we do in PIE  and CRUMB tests

Reply:
A solution here is to have and addrB1, addrB2, and addrB3 to use to test one on the selection modes.

Reply:
why not?? use `CAT` to test both selectionMode
```
await container.call('acc2acc', [tokenAddrC, {[addrC1]:`${58}@CAT`}])  // forward
await container.call('acc2acc', [tokenAddrC, {[addrC2]:`${3}@CAT`}])  // crumbs
await container.call('acc2acc', [tokenAddrC, {[addrC3]:`${103}@CAT`}]) // pie
```

Reply:
Okay, sorry my mistake it will work since we wont be testing forward.